From 25fb5f0f0a73bf7a47d6f6581b7f32c63e7e3605 Mon Sep 17 00:00:00 2001
From: Frank Hunleth <fhunleth@troodon-software.com>
Date: Sun, 3 Jun 2018 11:33:53 -0400
Subject: [PATCH] Add MMC environment location

Nerves stores its U-boot environment in a different location than the
default BBB builds, so add it in.
---
 include/configs/am335x_evm.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/configs/am335x_evm.h b/include/configs/am335x_evm.h
index e55736e..7c8e836 100644
--- a/include/configs/am335x_evm.h
+++ b/include/configs/am335x_evm.h
@@ -389,6 +389,9 @@
 #define CONFIG_ENV_OFFSET		0x001c0000
 #define CONFIG_ENV_OFFSET_REDUND	0x001e0000
 #define CONFIG_SYS_ENV_SECT_SIZE	CONFIG_SYS_NAND_BLOCK_SIZE
+#elif defined(CONFIG_ENV_IS_IN_MMC)
+#define CONFIG_SYS_MMC_ENV_DEV		0
+#define CONFIG_ENV_OFFSET		0x00100000
 #endif
 
 /* SPI flash. */
-- 
2.7.4

